app/pages/dashboard/PatientDrawer/useAgpCGM/buildGenerateAGPImagesFunction.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-undef */
+
+import _ from 'lodash';
+import { utils as vizUtils } from '@tidepool/viz';
+import Plotly from 'plotly.js-basic-dist-min';
+import * as actions from '../../../../redux/actions';
```

Comment:
This function is a rip of [this class method](https://github.com/tidepool-org/blip/blob/a95a74b3443f3f83e29491b779c0a4ea5edad8e5/app/pages/patientdata/patientdata.js#L922-L964) in PatientData.js.

I apologize for using a funky pattern here without checking in with the group, but essentially what I'm doing here is wrapping the main function in 2 layers of functions. The outer layers allow for a pseudo-dependency injection instead of having the function import the dependencies (vizUtils and Plotly) directly. In the test file it makes it much easier to mock vizUtils and Plotly.